### PR TITLE
Build: Upgrade chromatic

### PR DESCRIPTION
**Pull request by ndelangen**
upgrade chromatic

### CODE REVIEW

**Comment by shilman** [comment]
@ndelangen Do you know why the snapshots diffs are happening?

**Comment by ndelangen** [comment]
No I do not...

**Comment by ndelangen** [comment]
Before:
https://www.chromaticqa.com/component?appId=5a375b97f4b14f0020b0cda3&name=Addons%7CInfo%2FParameters&buildNumber=12007&specName=Using%20Parameters%20Across%20All%20Stories

After:
https://www.chromaticqa.com/component?appId=5a375b97f4b14f0020b0cda3&name=Addons%7CInfo%2FParameters&mode=interactive&buildNumber=12008&specName=Using%20Parameters%20Across%20All%20Stories&viewport=1200&browser=chrome

Difference:
Screenshot 2019-10-01 at 19 15 08

The diff also shows up in the canvas in chromatic... I'm puzzled

File: yarn.lock [hunk 1]
```diff
@@ -3765,6 +3765,19 @@
     memoizerific "^1.11.3"
     qs "^6.6.0"
 
+"@storybook/router@^5.2.1":
```
Comment (ndelangen) [inline yarn.lock]: This might be significant

File: yarn.lock [hunk 2]
```diff
@@ -1532,7 +1532,7 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
-"@babel/runtime@^7.0.0", "@babel/runtime@^7.1.2", "@babel/runtime@^7.2.0", "@babel/runtime@^7.3.1", "@babel/runtime@^7.3.4", "@babel/runtime@^7.4.2", "@babel/runtime@^7.4.3", "@babel/runtime@^7.4.4", "@babel/runtime@^7.4.5", "@babel/runtime@^7.5.0", "@babel/runtime@^7.5.5":
+"@babel/runtime@^7.0.0", "@babel/runtime@^7.1.2", "@babel/runtime@^7.2.0", "@babel/runtime@^7.3.4", "@babel/runtime@^7.4.2", "@babel/runtime@^7.4.3", "@babel/runtime@^7.4.4", "@babel/runtime@^7.4.5", "@babel/runtime@^7.5.0", "@babel/runtime@^7.5.5", "@babel/runtime@^7.6.2":
```
Comment (ndelangen) [inline yarn.lock]: this might be significant

File: yarn.lock (hunk 1 above)
Comment (ndelangen) [reply yarn.lock]: I deleted this

**Comment by ndelangen** [comment]
When I run locally, The space is there, and the acceptance stories work.

**Comment by ndelangen** [comment]
@shilman I now know.

It's a babel / eslint / prettier change that made the space be removed.
The chromatic-cli alpha is using latest and greatest stuff, so that got into monorepo root node_modules and that make the snapshots different.

What I did was:
separately made an upgrades PR, verified it made the same impact. (I had to resolve some eslint changes).
I upgraded to storybook-chromatic 2.2.2 first.
On this version the symbolic linked 'built-storybook' was also already broken.
So that means my debug version wasn't causing that either.

I fixed that by using the prebuild examples script I wrote for now.sh for chromatic.

That just should just leave a few flaky acceptance test, the usual.

I'm confident now that The package is in a releasable state.

**Pull request merged**